bowtie/_cli.py

```diff
@@ -970,22 +970,33 @@ def validate(
     "-l",
     "languages",
     type=click.Choice(sorted(KNOWN_LANGUAGES), case_sensitive=False),
-    callback=lambda _, __, value: frozenset(  # type: ignore[reportUnknownLambdaType]
-        LANGUAGE_ALIASES.get(each, each)  # type: ignore[reportUnknownArgumentType]
-        for each in value or KNOWN_LANGUAGES  # type: ignore[reportUnknownArgumentType]
+    callback=lambda _, __, value: (  # type: ignore[reportUnknownLambdaType]
+        KNOWN_LANGUAGES
```

Comment:
Ig its not really equivalent since we no longer put `KNOWN_LANGUAGES` through the for loop so they won’t get aliased and only then we would be able to compare `languages == KNOWN_LANGUAGES`

Reply:
If they get aliased then the equality won’t work.

Reply:
Ig its not really equivalent since we no longer put KNOWN_LANGUAGES through the for loop so they won’t get aliased and only then we would be able to compare languages == KNOWN_LANGUAGES

Reply:
If they get aliased then the equality won’t work.

Reply:
Did this make sense @Julian ?

Reply:
I'll look and see what you mean, but I'd expect there to be two "things" -- "all the known languages" and then "a mapping of aliases to canonical names", and checking whether we are not filtering languages should basically mean checking whether the languages we're using are "all the known languages" *after* de-aliasing, because at that point all of them should be being referred to via their canonical name.

Reply:
Oh I see, at the minute `KNOWN_LANGUAGES` isn't the "all known languages with their canonical names", because it's also being used for the `Choice`.

Reply:
So are you fine with this way ?

Reply:
I tried this but it doesn't really work since returning `None if not value` is not really the ideal way for e.g. if someone executes
`bowtie filter-implementations -d 2020` 

then in this case `if not value` is **True** so we should be defaulting to returning all the `KNOWN_LANGUAGES` (for comparison purposes (languages == KNOWN_LANGUAGES) and not even aliasing it) and not return `None` here.

Reply:
Got it, ok, then let's leave it for now.